### DISCUSSION TITLE
General discussion of flask swagger generator

### DISCUSSION THREAD

**Discussion by MDUYN** [coding-kitties/flask-swagger-generator#32] [General]
## 👋 Welcome to the flask swagger generator discussion!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MDUYN** [comment]
Hey, I am starting this discussion because I see a lot of interest in the project recently. @Chirema @Oracking @fizaashraf37, you are all added as maintainers of the project. For the past year there was not much work done on the project, however, I think the following points could really improve it:

- General improvements
  - Documentation website (will be initiated by coding kitties members).
  - Code of conduct / Working agreement
  - Release explanation
- Features
  - Nested schema support #28 
  - Custom tags to group endpoints 
  - Server url support #24
  - Composable generator classes (@Oracking request) #9
  - query params support #21
- Fixes
  - Refactor of repo architecture ("specificiers" need to renamed to "component" and each component should have its own module) #33 
  - Group does not work for endpoints without classes #26 
  - Support path input type #7 

Lets keep this discussion open, so we can update each-other on the points we are working on.

**Reply by MDUYN** [reply]
I am currently working on #33. Develop has already some of the fixes of @fizaashraf37 merged. In the coming days I will add various tests so we validate the work that has been done before releasing it.

**Reply by MDUYN** [reply]
@fizaashraf37 could you tell me if you have the right access policies now as a maintainer?

**Reply by fizaashraf37** [reply]
Hi @MDUYN , thanks for merging my PRs and yes I have the right access policies as maintainer.
